### PR TITLE
upgrade resolver and version

### PR DESCRIPTION
**Pull request by mut0u**
- update resolver version
- update package version

### CODE REVIEW

**Comment by UnkindPartition** [comment]
Why?

**Comment by mut0u** [comment]
When I use the newest resolver (lts-13.2),  the stack will search the hackage. I think the stack declare need upgrade the version to make sure that resolver lts-13.2 can use this package.

**Comment by UnkindPartition** [comment]
Sorry, I don't understand your explanation. I am going to close this. If you need to change stack.yaml, you can just do it locally, no need to contribute this change. If this fixes any particular issue, please give a detailed description of what the issue is.

**Comment by mrkkrp** [comment]
@mut0u `stack.yaml` is in not distributed with the package. So to outside world it'd look that you just bumped the version for no reason.
